Projects/DBS/DBS03/DBS03-AWS201.md

```diff
@@ -0,0 +1,53 @@
+# DBS03-AWS201 - SQLServer Native Backup and Restore on RDS
+
+## Cloud Service Provider
+
+- Amazon Web Services
+
+
+## Difficulty
+
+- Level 200 (Intermediate)
+
+
+## Project's Author(s)
+
+- [Jagan](https://twitter.com/JAG2wt)
+
+## Objectives
+
+### You need to complete the following:
+
+- Enable native backup and restore on RDS
+- Take native SQL Server backup to S3 bucket 
+- Restore native backup from S3 bucket
+
+
+### You need to answer the following:
+
+- How to migrate database from on-prem to RDS?
+- How to restore database from native SQL Server backup?
+- How to take manual backup on RDS?
+
+## References
+- [How to perform native backups of an Amazon RDS DB instance?](https://aws.amazon.com/premiumsupport/knowledge-center/native-backup-rds-sql-server/)
+- [Setting Up for Native Backup and Restore](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/SQLServer.Procedural.Importing.html#SQLServer.Procedural.Importing.Native.Enabling)
+- [Working With Backups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithAutomatedBackups.html)
+- [Running SQL Server Databases on Amazon RDS for SQL Server - AWS Virtual Workshop](https://youtu.be/twOglkIFbXU)
+- [Amazon RDS Free Tier](https://aws.amazon.com/rds/free/)
+
+## Costs
+
+- Included in the Free Tier
+
+
+## Estimated time to complete
+- 45-60 minutes
+
+
+## Tips
+- Having database knowledge is an added advantage
+- 750 hours of Amazon RDS Single-AZ `db.t2.micro` included in free tier

```

Comment:
This should be placed in the "Costs" section please

Reply:
You want me to move this line to "Costs" section?
> 750 hours of Amazon RDS Single-AZ `db.t2.micro` included in free tier

Please excuse me, am new to github